src/App.tsx

```diff
@@ -1,18 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Pagination } from './components/Pagination';
 import './App.css';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [selectedValue, setSelectedValue] = useState('5');
+  const [active, setActive] = useState(1);
+  const optionValue = ['3', '5', '10', '20'];
```

Comment:
Why can't you use just numbers here instead? 
```suggestion
  const [selectedValue, setSelectedValue] = useState(5);
  const [active, setActive] = useState(1);
  const optionValue = [3, 5, 10, 20];
```